layers/+lang/elixir/packages.el

```diff
@@ -91,10 +93,13 @@
       "ta" 'alchemist-mix-test
       "tb" 'alchemist-mix-test-this-buffer
       "tt" 'alchemist-mix-test-at-point
-      "tf" 'alchemist-mix-test-file
+      "tf" 'alchemist-project-run-tests-for-current-file
```

Comment:
Please note this breaking change... my goal was to make the key bindings more intuitive.